src/config.h

```diff
@@ -29,7 +29,7 @@
 
 // Edit to control what features Makefile'd raylib is compiled with
 #if defined(RAYLIB_CMAKE)
```

Comment:
I don't like mixing the build system with the library logic... is there any other alternative on CMake side to avoid this `RAYLIB_CMAKE`?

Reply:
@raysan5 There could be but I would suggest it to be in another pull request. This one is quite big and is mostly segmening the cmake files into more reasanable chunks.